test/integration/smoke/test_primary_storage_scope.py

```diff
@@ -0,0 +1,170 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+""" BVT tests for Primary Storage
+"""
+
+# Import System modules
+# Import Local Modules
+from marvin.cloudstackTestCase import *
+from marvin.lib.base import (Host, StoragePool, Cluster, updateStoragePool, changeStoragePoolScope)
+from marvin.lib.common import (get_zone, get_pod, list_clusters)
+from nose.plugins.attrib import attr
+
+class TestPrimaryStorageScope(cloudstackTestCase):
+
+    def setUp(self):
+
+        self.apiclient = self.testClient.getApiClient()
+        self.dbclient = self.testClient.getDbConnection()
+        self.services = self.testClient.getParsedTestDataConfig()
+        self.zone = get_zone(self.apiclient, self.testClient.getZoneForTests())
+        self.pod = get_pod(self.apiclient, self.zone.id)
+        self.debug("here")
+        self.debug(self.services)
+        self.cluster1 = list_clusters(self.apiclient)[0]
+        self.debug("here1")
+        self.debug(self.cluster1)
+        self.cluster = {
+            'clustername': 'C0_testScope',
+            'clustertype': 'CloudManaged'
+        }
+        self.cluster2 = Cluster.create(self.apiclient,
+                                       self.cluster,
+                                       zoneid=self.zone.id,
+                                       podid=self.pod.id,
+                                       hypervisor=self.cluster1.hypervisortype
+                                       )
+        self.storage = StoragePool.create(self.apiclient,
+                                          self.services["nfs"],
+                                          scope = 'ZONE',
+                                          zoneid=self.zone.id,
+                                          hypervisor=self.cluster1.hypervisortype
+                                          )
+        self.debug("Created storage pool %s in zone scope", self.storage.id)
+        return
+
+    def tearDown(self):
+        StoragePool.delete(self.storage, self.apiclient)
+        Cluster.delete(self.cluster2, self.apiclient)
```

Comment:
please look at the convention (a `cleanup` array) and implement thus.

Reply:
Done